submissions/ArthurGorbenko/js-dom/script.js

```diff
@@ -47,26 +47,28 @@ const puppies = [
 
 const image = document.querySelector(".info_image");
 const parameters = document.querySelectorAll(".parameters");
-
 const menu = document.querySelector(".menu");
-menu.addEventListener("click", function(event) {
-  let target = event.target;
-  while (target != menu) {
-    puppies.forEach(el =>
-      target.classList.contains(el.Breed) ? replace(target, el) : null
-    );
-    target = target.parentNode;
-  }
-});
+
+menu.addEventListener("click",doOnClick);
+
+function doOnClick(event){
```

Comment:
It unnecessary to correct it now but in future please select convenient names for function
To take an example `selectMenuItem` or something like that
Because many things in js working when you do a click))

Reply:
Good advice! Approve it ;)